File: 30days/06/06.py
#!/usr/bin/env python

n = int(input())

for i in range(0, n):
    mystring = input()
    even = ""
    odd = ""
    for i in range(0, len(mystring)):
        if i % 2 == 0:
            even += mystring[i]
        else:
            odd += mystring[i]
    print("%s %s" % (even, odd))

File: 30days/06/06.rb
#!/usr/bin/env ruby

n = gets.chomp.to_i

(1..n).each do
    mystring = gets.chomp
    even = ""
    odd = ""
    (0...mystring.length).each do |i|
        if i.even?
            even += mystring[i]
        else
            odd += mystring[i]
        end
    end
    puts "#{even} #{odd}"
end
